php/php-src
# Refactor `ext/pgsql` test

I refactored the `ext/pgsql` tests based on some recent pull requests.
The content and results remain unchanged. This is just an internal improvement.

With this refactoring, we can expect the following benefits:

* Test independence:
  Some tests were dependent on the execution order of previous and subsequent tests.
  All tests can now be executed independently.
* Test parallelism
  One test no longer interferes with another. All tests can be run in parallel.

Specifically, i made the following modifications: For readability, each modification is fixed in a separate commit.

* Use `--CLEAN--` to delete objects at the end of the test.
  https://github.com/php/php-src/commit/faabc3fba80f5b098373fb18cc3347c84b9f34e2
* Moved `*.inc` to another folder to make filenames easier to read.
  https://github.com/php/php-src/commit/7b507cd072babec6d0d75a88c31b418be4b9517c
* Tables used for tests are defined for each test instead of being set all at once in `config.inc`
  https://github.com/php/php-src/commit/002fe321e0bd10fbb31ebb6fcd0b1015ff1fb6e7 https://github.com/php/php-src/commit/293b0b01eacef73a7281dd988ffc82046e9148d6
* Use different object names for each test. The name is based on the test file name.
  https://github.com/php/php-src/commit/002fe321e0bd10fbb31ebb6fcd0b1015ff1fb6e7 https://github.com/php/php-src/commit/5ab45c70b9c5829493ece0f0b5cba522e4764e3d

Ref:
* https://github.com/php/php-src/pull/11855
* https://github.com/php/php-src/pull/11879
* https://github.com/php/php-src/pull/12555
